src/utils/chain.ts

```diff
@@ -87,6 +88,20 @@ export async function queryMinDelegatorStake(): Promise<u128> {
   return api.consts.parachainStaking.minDelegatorStake as u128
 }
 
+export async function queryStakingRates(): Promise<StakingRatesChain> {
+  const api = await getConnection()
+  return api.call.staking.getStakingRates<StakingRatesChain>()
+}
+
+export async function getUnclaimedStakingRewards(
+  account: string
+): Promise<Balance> {
+  const api = await getConnection()
+  return api.call.staking.getUnclaimedStakingRewards<Balance>(
+    account
+  )
+}
```

Comment:
We need to check if `api.call.staking` is defined. Nodes might not have this call immediately.

Reply:
[55750f6](https://github.com/BTE-Trusted-Entity/stakeboard/pull/74/commits/55750f61359435c1de6f81b30e7ee26f2af52dc5)